### ISSUE TITLE
Panel sometimes gets pushed underneath gutter

### ISSUE THREAD

**Issue by weltschmerz1** [nathancahill/Split.js#73]
I haven't tested this much, but it seems that depending on the gutter width and the window width, sometimes the panel width plus gutter width adds up to more than 100%, which ends up pushing the panel underneath the gutter.

Consider a case with two panels that are split next to each other. For me, this has happened only when collapsing the left panel all the way to 0. Curiously, collapsing the right pane completely did not reproduce the issue.

Initial state:

```
                     ↓ gutter
xxxxxxxxxxxxxxxxxx   |   xxxxxxxxxxxxxxxxxx
x                    |                    x
x    left panel      |      right panel   x
x                    |                    x
xxxxxxxxxxxxxxxxxx   |   xxxxxxxxxxxxxxxxxx
```

Then I drag the gutter all the way to the left, which hides the left panel.

Desired result:

```
↓ gutter (left panel hidden)
| xxxxxxxxxxxxxxxxxx
|                  x
|  right panel     x
|                  x
| xxxxxxxxxxxxxxxxxx
```

Actual result:

```
↓ gutter (left panel hidden)
|
|
|
|
|
xxxxxxxxxxxxxxxxxx
                 x
   right panel   x         (right panel pushed underneath gutter)
                 x
xxxxxxxxxxxxxxxxxx
```

The thing about the gutter and the right panel adding up to more than 100% width is just my initial intuition and it could obviously be something else altogether.

For now, my temporary solution was to set the gutter width to one pixel smaller using css:

```css
.gutter.gutter-horizontal {
  width: 9px !important;
}
```

**Comment by nathancahill** [comment]
Thanks for the excellent bug report. Is this using the latest version (1.2.0)?

**Comment by sanleong** [comment]
I confronted this situation. Just in case somebody else has the issue, a workaround for this behavior is to declare the sides of the panes to something less than 100%.
i.e. `splitPaneObj = Split(['#pane1', '#pane2', '#pane3'], {
                            sizes: [20, 39.96, 39.96],
                            minSize: 0,
                        })
                        }`

**Comment by dchacke** [comment]
@nathancahill Yeah it's on 1.2.0.

**Comment by nathancahill** [comment]
Ok, thanks, I was able to reproduce. Will have a fix shortly. Sorry about that.

**Comment by nathancahill** [comment]
Haven't been able to find a clean solution to this. It tricky because of the floating-point math when dividing by 3 and summing up to 100%. Will keep working on it.

**Comment by sanleong** [comment]
Thanks for the feedback.
Really appreciated.

Regards,
Gilberto Leon

On Apr 12, 2017, at 12:44 AM, Nathan Cahill <notifications@github.com<mailto:notifications@github.com>> wrote:


Haven't been able to find a clean solution to this. It tricky because of the floating-point math when dividing by 3 and summing up to 100%. Will keep working on it.

**Comment by nathancahill** [comment]
This issue should be fixed, available here if you mind testing the latest build in the repo, this file: https://github.com/nathancahill/Split.js/blob/master/split.js or if you're including it via CDN: https://rawgit.com/nathancahill/Split.js/master/split.js

Otherwise it will be released after testing as a new version on NPM.

**Comment by nathancahill** [comment]
Released in 1.3.5. Thanks!
